src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Calibrations;
+import frc.robot.Constants;
+import frc.robot.RobotMap;
+
+import java.util.TreeMap;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+
+public class Lift extends SubsystemBase {
+  private CANSparkMax elevator = new CANSparkMax(RobotMap.ELEVATOR_MOTOR_CAN_ID, MotorType.kBrushless);
+
+  private final RelativeEncoder elevatorEncoder;
+  private final RelativeEncoder armEncoder;
+
+  private SparkMaxPIDController elevatorPID =  elevator.getPIDController();
+
+  private CANSparkMax arm = new CANSparkMax(RobotMap.ARM_MOTOR_CAN_ID, MotorType.kBrushless);
+  private SparkMaxPIDController armPID =  arm.getPIDController();
+
+  TreeMap<LiftPosition, Pair<Double, Double>> liftPositionMap;
```

Comment:
This needs a comment to say what this Map is. It indicates the elevator and arm positions associated with each position of the lift. We should be super explicit about what the first value and second value refer to

Reply:
ok, I also added the units